tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -218,13 +218,32 @@ shader, this value can be used as desired.
 | out vec4 **CUSTOM3**                   |                                                        |
 +----------------------------------------+--------------------------------------------------------+
 
+.. note::
+
+    MODELVIEW_MATRIX combines both the the MODEL_MATRIX and VIEW_MATRIX and is better suited when floating point issues may arise
```

Comment:
```suggestion
    ``MODELVIEW_MATRIX`` combines both the the ``MODEL_MATRIX`` and ``VIEW_MATRIX`` and is better suited when floating point issues may arise. For example, if the object is very far away from the world origin, you may run into floating point issues when using the separated ``MODEL_MATRIX`` and ``VIEW_MATRIX``.
```

Copying this over from my previous review comment as it appears to have been missed